src/test/java/com/google/cloud/tools/maven/RunMojoTest.java

```diff
@@ -50,6 +51,11 @@
 @RunWith(JUnitParamsRunner.class)
 public class RunMojoTest extends AbstractDevServerTest {
 
+  private static final String STANDARD_PROJECT_WEBAPP =
```

Comment:
just need to add a negative test here, where it fails (one standard app, one non-standard app)

Reply:
👍